client/app/src/services/network.service.js

```diff
@@ -17,6 +17,10 @@
     var ark = require('../node_modules/arkjs')
     ark.crypto.setNetworkVersion(network.version || 23)
 
+    const Moment = require('moment-timezone')
+    const MomentRange = require('moment-range')
+    const moment = MomentRange.extendMoment(Moment)
```

Comment:
It would be good if you could rename one of the "moment" constants to make them unique. I know it's case sensitive, but it may confuse people.